docs/source/examples/pt_zne.md

```diff
@@ -0,0 +1,378 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+```{tags} cirq, zne, pt, basic
+```
+# ZNE with Noise Tailoring
+
+This tutorial helps us understand the cases when noise tailoring is helpful for improving the performance of 
+quantum error mitigation techniques. 
+
+We will
+
+- compare coherent noise to incoherent noise
+- Pauli twirl coherent noise to be incoherent
+- compare the performance of [Zero-Noise Extrapolation](../guide/zne.md) (ZNE)
+on its own and in combination with [Pauli Twirling](../guide/pt.md) (PT) when the circuit is subjected to coherent noise
+
+## Coherent noise vs. Incoherent noise
+
+**Coherent noise** is a reversible process as long as the noisy unitary transformation is known beforehand which is not always the case. These types of noise maintain the purity of the state. But in a quantum circuit, they are easily carried across the circuit such that they scale up quadratically. Dealing with coherent noise requires a large resource overhead.  
```

Comment:
> But in a quantum circuit, they are easily carried across the circuit such that they scale up quadratically.

How much work would this be to make a little more precise? "They scale up quadratically" in particular.

Reply:
Requires some math to show this.

Reply:
Okay lets leave that out, but it might be good to specify what "they" are.